scripts/initiate_ritual.py

```diff
@@ -35,36 +33,76 @@
     type=click.Choice(["GlobalAllowList", "OpenAccessAuthorizer", "ManagedAllowList"]),
     required=True,
 )
-def cli(domain, duration, network, account, access_controller):
+@click.option(
+    "--fee-model",
+    "-f",
+    help="The name of a fee model contract.",
+    type=click.Choice(["FreeFeeModel", "BqETHSubscription"]),
+    required=True,
+)
+@click.option(
+    "--num-nodes",
+    help="Number of nodes to use for the ritual.",
+    type=int,
+    required=False,
```

Comment:
Should this be required?

Reply:
`--num-nodes` and `--handpicked` conflict since number of nodes is only used for sampling.  I included some enforcement of this below.